tests/test_launcher.py

```diff
@@ -204,6 +207,67 @@ def test_watchdog_launch(monkeypatch):
     pyfluent.launch_fluent(start_watchdog=True)
 
 
+def test_fluent_launchers():
+    if not check_docker_support() and not pypim.is_configured():
+        standalone_meshing_launcher = create_launcher("standalone", mode="meshing")
+        standalone_meshing_session = standalone_meshing_launcher()
+        assert standalone_meshing_session
+        assert standalone_meshing_launcher.start_transcript is True
+
+        standalone_meshing_session_2 = standalone_meshing_launcher(
+            start_transcript=False
+        )
+        assert standalone_meshing_session_2
+        assert standalone_meshing_launcher.start_transcript is False
```

Comment:
@seanpearsonuk Testing standalone session from previous launcher but after changing default value of `start_transcript`. Here we are checking kwargs passed in `__call__` method of standalone launcher.